server/certs/server.crt

```diff
@@ -1,20 +0,0 @@
------BEGIN CERTIFICATE-----
```

Comment:
revert these for local testing, but keep alternates. Alternatively, could store these in cloud storage or kubernetes secrets.